### DISCUSSION TITLE
Related to schema conversion

### DISCUSSION THREAD

**Discussion by shubhambajad** [apideck-libraries/portman#327] [Q&A]
Hi Guys,

I have created a schema file which consist of support for multiple payload i.e. application/json, x-www-urlencoded. 
But when I created a collection using a portman tool collection gets generated with x-www-urlencoded body only so is that intended functionality or we have to do some configuration so that it will convert it into application/json body?

Because for every request I am able to see  x-www-urlencoded body not a json body.

**Comment by thim81** [comment]
@shubhambajad Could you share your OpenAPI specification?

Since this seems like a problem, we should be able to resolve.
Portman currently supports only `application/json` payloads (a feature request for [x-www-urlencoded](https://github.com/apideck-libraries/portman/issues/325 is open).

We made Portman a bit smarter in the upcoming release to handle `application/json` or variants of JSON, but your case might be something we could solve as well.
